projects/arrays/text-processor/src/solution-alternate.ts

```diff
@@ -36,18 +36,21 @@ const aligners = {
 
 		return remainingSpaces
 			? [
-					Array.from({ length: Math.floor(remainingSpaces / 2) }, () => " "),
+					" ".repeat(Math.floor(remainingSpaces / 2)),
 					line,
-					Array.from({ length: Math.ceil(remainingSpaces / 2) }, () => " "),
-			  ].join(" ")
+					" ".repeat(Math.ceil(remainingSpaces / 2)),
+			  ].join("")
 			: line;
 	},
-	right: (line: string, width: number) => line.padEnd(width),
+	right: (line: string, width: number) => line.padStart(width),
 };
 
 function alignLines(
 	lines: string[],
 	{ align = "left", width }: AlignmentOptions
 ) {
-	return lines.map(aligners[align], width);
+	const aligner = aligners[align];
+	return lines.map((line, index, lines) => {
+		return aligner(line, width);
+	});
```

Comment:
I'm just nitpicking, but in the name of simplicity I think we can get this down to one line or so:

```suggestion
	return lines.map((line) => aligners[align](line, width));
```

Reply:
Everything is working so I'll push this up now 👍